src/abstractdataframe/selection.jl

```diff
@@ -469,8 +479,52 @@ SELECT_ARG_RULES =
     It is not allowed to pass `renamecols=false` if `old_column` is empty
     as it would generate an empty column name.
 
-    Column renaming and transformation operations can be passed wrapped in
-    vectors (this is useful when combined with broadcasting).
+    # Rules when `new_column_name` is a vector of `Symbol` or a string or is `AsTable`
+
+    In this case it is assumed that `fun` returns multiple columns.
+
+    If `fun` returns one of `AbstractDataFrame`, `NamedTuple`, `DataFrameRow`,
+    `AbstractMatrix` then rules described below for `args` being a function or
+    a type apply.
```

Comment:
Maybe put this at the end so that it's clearer what "below" means.

Reply:
I made it more precise what we reference to